src/components/HomePage/MainContent.js

```diff
@@ -8,6 +8,9 @@ import './Homepage.css';
 export const MainContent = ({setModalOpen}) => {
     const [title, setTitle ] = useState('');
     const [content, setContent ] = useState('');
+    const [label, setLabel] = useState('');
+    const [priority, setPriority] = useState('low');
+
```

Comment:
when you get to use more than 2 usestates, consider using reducer for the same.

Reply:
Yes, I will do this when refactoring